frontend/src/AlertsManagement/AlertInfo.js

```diff
@@ -46,14 +50,30 @@ class AlertInfo extends Component {
     newAlert.status = statusToChangeTo;
     this.setState({ alert: newAlert });
   }
+
+  handlePriorityChange = (newPriority) => {
+    const { alert } = this.state;
+
+    // We have to get the P0, P1, or P2 version to match with enum representation in backend.
+    const numToEnum = Object.keys(priorityLevels)[Object.values(priorityLevels).indexOf(newPriority)];
```

Comment:
i was a bit confused on what is this `numToEnum` does. so you need to do the mapping with the map to convert it back?

seems the priority is just the "0", "1", "2" (the numerical value) etc for easier sorting?

Reply:
Yup! It's for easier sorting, but to translate it back to the backend (so that the backend understands 0, 1, 2) should convert it back. If I didn't do it in the frontend, I think I would have to do it in the backend anyway